flow/size.yaml

```diff
@@ -62,3 +83,111 @@ components:
           type: integer
           default: 1518
           x-field-uid: 2
+    Flow.SizeWeightPairs:
+      type: object
+      description: |-
+        Frame size distribution, defined as <size, weight> pairs (including IMIX distribution).
```

Comment:
Any comment on repeatability of randomness ?

Reply:
The reason for asking this is, let's assume we have a following distribution:
- 64B => 2
- 128B => 3

Possible sequences could be:
1. Exhaustive (Total packets 10): 64 64 64 64 128 128 128 128 128 128
2. Repeat after minimum exhaustion (Total packets: 10): 64 64 128 128 128 64 64 128 128 128
3. Uniformly distributed (Total packets: 3): 64 128 128
4. Non-uniform (Total packets 10): 64 128  64 64 64 128 128 128 128 128

Reply:
Weight pair distribution only ensures the distribution ratio will be maintained over a fair number of packets. The sequence is not guaranteed.